src/styles/header.scss

```diff
@@ -0,0 +1,67 @@
+@use './variables.scss' as *;
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  text-transform: uppercase;
+  width: auto;
+  background-color: #fff;
+  padding-left: 50px;
+  padding-right: 50px;
```

Comment:
The padding-left and padding-right are set to 50px. Consider using a more responsive unit like percentages or rems to ensure the layout adapts better to different screen sizes.